Source/Actors/Player.cs

```diff
@@ -1,2364 +1,2377 @@
-			return true;
-		}
-
-		return false;
-	}
-
-	public void Stop() => velocity = Vec3.Zero;
-
-	#endregion
-}
+using Celeste64.TAS;
+using Celeste64.TAS.Input.Commands;
+
+namespace Celeste64;
+
+/// <summary>
+/// Welcome to the monolithic player class! This time only 2300 lines ;)
+/// </summary>
+public class Player : Actor, IHaveModels, IHaveSprites, IRidePlatforms, ICastPointShadow
+{
+	#region Constants
+
+	private const float Acceleration = 500;
+	private const float PastMaxDeccel = 60;
+	private const float AirAccelMultMin = .5f;
+	private const float AirAccelMultMax = 1f;
+	private const float MaxSpeed = 64;
+	private const float RotateThreshold = MaxSpeed * .2f;
+	private const float RotateSpeed = MathF.Tau * 1.5f;
+	private const float RotateSpeedAboveMax = MathF.Tau * .6f;
+	private const float Friction = 800;
+	private const float AirFrictionMult = .1f;
+	private const float Gravity = 600;
+	private const float MaxFall = -120;
+	private const float HalfGravThreshold = 100;
+	private const float JumpHoldTime = .1f;
+	private const float JumpSpeed = 90;
+	private const float JumpXYBoost = 10;
+	private const float CoyoteTime = .12f;
+	private const float WallJumpXYSpeed = MaxSpeed * 1.3f;
+
+	private const float DashSpeed = 140;
+	private const float DashEndSpeedMult = .75f;
+	private const float DashTime = .2f;
+	private const float DashResetCooldown = .2f;
+	private const float DashCooldown = .1f;
+	private const float DashRotateSpeed = MathF.Tau * .3f;
+
+	private const float DashJumpSpeed = 40;
+	private const float DashJumpHoldSpeed = 20;
+	private const float DashJumpHoldTime = .3f;
+	private const float DashJumpXYBoost = 16;
+
+	private const float SkidDotThreshold = -.7f;
+	private const float SkiddingStartAccel = 300;
+	private const float SkiddingAccel = 500;
+	private const float EndSkidSpeed = MaxSpeed * .8f;
+	private const float SkidJumpSpeed = 120;
+	private const float SkidJumpHoldTime = .16f;
+	private const float SkidJumpXYSpeed = MaxSpeed * 1.4f;
+
+	private const float WallPushoutDist = 3;
+	public const float ClimbCheckDist = 4; // TAS: publicized
+	private const float ClimbSpeed = 40;
+	private const float ClimbHopUpSpeed = 80;
+	private const float ClimbHopForwardSpeed = 40;
+	private const float ClimbHopNoMoveTime = .25f;
+
+	private const float SpringJumpSpeed = 160;
+	private const float SpringJumpHoldTime = .3f;
+
+	private const float FeatherStartTime = .4f;
+	private const float FeatherFlySpeed = 100;
+	private const float FeatherStartSpeed = 140;
+	private const float FeatherTurnSpeed = MathF.Tau * .75f;
+	private const float FeatherAccel = 60;
+	private const float FeatherDuration = 2.2f;
+	private const float FeatherExitXYMult = .5f;
+	private const float FeatherExitZSpeed = 60;
+
+	public const float FreecamRotateSpeed = 0.5f; // TAS: publicized
+	public const float FreecamZoomSpeed = 5.0f; // TAS: publicized
+	public const float FreecamMoveSpeed = 3.0f; // TAS: publicized
```

Comment:
You don't need to annotate them as publicized, since they're our constants.
Tho it might make sense to just mark them as from the mod?